internal/os_updater/emt/download.go

```diff
@@ -216,18 +220,15 @@ func readJWTToken(fs afero.Afero, path string) (string, error) {
 	return strings.TrimSpace(string(token)), nil
 }
 
-
 // checkDiskSpace checks if there is enough disk space to download the artifacts.
```

Comment:
func name in docstring should be isDiskSpaceAvailable

Reply:
fixed.